addons/hr/views/hr_views.xml

```diff
@@ -17,830 +16,89 @@
             sequence="0"/>
 
         <menuitem
-            id="hr_menu_hr_reports"
-            name="Reporting"
+            id="menu_hr_employee_payroll"
+            name="Payroll"
             parent="menu_hr_root"
-            sequence="95"/>
+            groups="group_hr_user"
+            sequence="3"/>
 
-        <menuitem
-            id="menu_hr_reporting_timesheet"
-            name="Reporting"
-            parent="menu_hr_root"
-            groups="group_hr_manager,group_hr_user"
-            sequence="99"/>
+            <menuitem
+                id="menu_hr_employee_user"
+                name="Employees"
+                action="open_view_employee_list_my"
+                parent="menu_hr_employee_payroll"
+                sequence="1"/>
 
         <menuitem
-            id="menu_human_resources_configuration"
-            name="Configuration"
+            id="menu_hr_employee"
+            name="Company Address"
```

Comment:
So "Company Address" is the name of the menu to open the employee directory? :see_no_evil:

Reply:
@odony It has been carefully pondered by our team of dedicated product owners.

I don't have the key to the wisdom, they do.

Ping @RomainLibert @tde-banana-odoo

Reply:
I have the key to the closet if you wish.